### ISSUE TITLE
allows some bad input?

### ISSUE THREAD

**Issue by cainus** [cainus/hottap#5]
```
 TypeError: Cannot read property 'body' of undefined
  at /Users/sdecesare/code/node/percolator/node_modules/hottap/hottap.js:161:19
  at ClientRequest.<anonymous> (/Users/sdecesare/code/node/percolator/node_modules/hottap/hottap.js:117:5)
  at ClientRequest.emit (events.js:67:17)
  at Socket.<anonymous> (http.js:1127:11)
  at Socket.emit (events.js:67:17)
  at Array.0 (net.js:319:25)
  at EventEmitter._tickCallback (node.js:192:40)
```

**Comment by cainus** [comment]
line 137 should be: !!response && !!response.body

**Comment by cainus** [comment]
fixed in 1.0.1
